webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+
+const frontEndStaticDir = path.resolve(__dirname, 'critiquebrainz/frontend/static');
+
+module.exports = (env, argv) => {
+  const isProd = argv.mode == 'production';
+  return {
+    mode: argv.mode,
+    entry: {
+      'main': path.resolve(frontEndStaticDir, 'scripts/main.js'),
+      'common': path.resolve(frontEndStaticDir, 'scripts/common.js'),
+      'leaflet': path.resolve(frontEndStaticDir, 'scripts/leaflet.js'),
+      'rating': path.resolve(frontEndStaticDir, 'scripts/rating.js'),
+      'spotify': path.resolve(frontEndStaticDir, 'scripts/spotify.js'),
+      'wysiwyg-editor': path.resolve(frontEndStaticDir, 'scripts/wysiwyg-editor.js'),
+    },
+    output: {
+      path: path.resolve(__dirname, 'critiquebrainz/frontend/static/build/'),
```

Comment:
could this be path.resolve(frontEndstaticDir, build) in order to not duplicate path?

Reply:
Done!